platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/cache/CookieCache.java

```diff
@@ -58,9 +58,7 @@ public void addActiveSp(String key, String activeSp) {
             LOGGER.error("Cannot add sp [{}] for [{}]: does not exist", activeSp, key);
             return;
         }
-        synchronized (dataWrapper) {
```

Comment:
No. All of the synchronized blocks on the `dataWrapper` instances need to remain in this class. And as you'll be synchronizing, you won't need the `ConcurrentHashSet`. Revert this file.